tests/userTests.py

```diff
@@ -53,7 +53,7 @@ def setUpToken(self,email):
     def test_registration_no_token(self):
         """Test without token."""
         self.logout()
-        postJson = {"email": "user@agency.gov", "name": "user", "agency": "agency", "title": "title", "password": "userPass"}
+        postJson = {"email": "user@agency.gov", "name": "user", "cgac_code": "agency", "title": "title", "password": "userPass"}
```

Comment:
Very minor nit: changing the word "agency" to "000", "999" or similar in this fake JSON would make it more explicit that CGAC code doesn't represent an agency name.